modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/package.scala

```diff
@@ -31,7 +31,9 @@ package object server {
   }
 
   implicit class SeqActionOps[A](a: SeqAction[A]) {
-    def runSeqAction: TrySeq[A] = a.run.unsafePerformSyncAttempt.leftMap[SeqexecFailure](SeqexecException).join
+    def runSeqAction: TrySeq[A] = a.run.handle{
+      case e: Exception => SeqexecException(e).left
+    }.unsafePerformSync
```

Comment:
My impression is that having the `unsafe...` call here is a bit problematic for callers which don't know that you do. I'd rather remove it or rename the method to `unsafeRunSeqAction`. WDYT?